### ISSUE TITLE
Please enhance method  `cxPartialMached`

### ISSUE THREAD

**Issue by superhxl** [DEAP/deap#472]
The method cxPartialMached seems only works for sequence like [0, 1, ..., N-1]. Is it possible for all kinds of sequence?
![](https://i.loli.net/2020/03/29/gIJhiO4suorw2Vm.png)

**Comment by fmder** [comment]
Yup, making it work on otherwise would add unnecessary complexity.
